src/lib/themes/index.js

```diff
@@ -0,0 +1,28 @@
+import defaultsDeep from 'lodash.defaultsdeep';
+
+import darkMode from './dark-mode';
+import highContrast from './high-contrast';
+import defaultColors from './default-colors';
+
+const mergeWithDefaults = colors => defaultsDeep({}, colors, defaultColors);
+
+const themeMap = {
+    'dark-mode': mergeWithDefaults(darkMode),
+    'high-contrast': mergeWithDefaults(highContrast),
+    'standard': defaultColors
+};
```

Comment:
I'm open to better names for our themes (ex standard / original / default / ???). This is easy to modify later too.

Right now the color data structure match the payload that is passed to `scratch-blocks`. I could see this being augmented later if we have some properties that are part of the editor but not in the blocks.

Reply:
I think these names are fine for now, until/unless we settle on different user-facing names later.

> I could see this being augmented later if we have some properties that are part of the editor but not in the blocks.

Makes sense! Seems like it would be pretty easy to turn it into `theme = { blocksColors: {}, otherStuff: {} }` if necessary.